remoteworkitem/github.go

```diff
@@ -35,7 +36,8 @@ func (f *githubIssueFetcher) listIssues(query string, opts *github.SearchOptions
 func (g *GithubTracker) Fetch() chan TrackerItemContent {
 	configuration, err := config.GetConfigurationData()
 	if err != nil {
-		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+		fmt.Print(errors.Wrap(err, "Failed to setup the configuration"))
```

Comment:
Maybe have `Fetch()` return `(chan TrackerItemContent, <-chan error)` or something alike to allow for proper error checking. Printing the error won't help.